streams-tests/shared/src/test/scala/zio/stream/ZStreamSpec.scala

```diff
@@ -2308,6 +2308,20 @@ object ZStreamSpec extends ZIOBaseSpec {
             }
           )
         ),
+        suite("scan")(
+          testM("scan")(
+            assertM(
+              ZStream(1, 2, 3, 4, 5).scan(0)(_ + _).runCollect
+            )(equalTo(Chunk(0, 1, 3, 6, 10, 15)))
+          )
+        ),
+        suite("scanReduce")(
+          testM("scanReduce")(
+            assertM(
+              ZStream(1, 2, 3, 4, 5).scanReduce(_ + _).runCollect
+            )(equalTo(Chunk(1, 3, 6, 10, 15)))
+          )
+        ),
```

Comment:
Could we write these as property checks that verify equivalence to Chunk#scanLeft?

Reply:
Sure, will update later today

Reply:
@iravid updated